docs/setup.md

```diff
@@ -1,43 +1,9 @@
 # Setup the Provisioner machine with docker-compose.yml file
 
-## Install git and git lfs as follows
+Execute the following commands to bring up the Tinkerbell stack and setup the provisioner:
+```shell
+$ wget https://raw.githubusercontent.com/infracloudio/tink/deploy_stack/setup.sh && chmod +x setup.sh
+$ ./setup.sh
+```
 
-1. ### Setup git and git lfs
```

Comment:
Is it worth retaining the manual instructions for local development? cc @nathangoulding